test_runner/cloud_regress/README.md

```diff
@@ -3,19 +3,35 @@
 * Create a Neon project on staging.
 * Grant the superuser privileges to the DB user.
 * (Optional) create a branch for testing
-* Configure the endpoint by updating the control-plane database with the following settings:
+* Configure the default endpoint for the project using the admin interface with the following settings:
```

Comment:
I suggest adding here that it should go to the `pg_settings` field of the json, just in case.